Makefile

```diff
@@ -131,6 +131,10 @@ delete-spire:
 	done
 	helm delete spire -n nsm
 
+# RBAC for registry-k8s
+rbac-for-registry-k8s:
```

Comment:
Users will have to deploy the resources required by the k8s-registry by themselves?

Reply:
Not really @LionelJouin. I guess you're talking about the RBAC resources, am I right? So I'm trying to keep those separated from the operator. Both RBAC and CRDs should be installed from yaml manifests or via Operator Lifecycle Manager (OLM). We shouldn't give permissions to an operator to manage RBAC. It's not a good design keeping security first. So what we'll do is to provide a single yaml file that installs all resources including the operator at once. I don't want users having to run make targets for install. Only developers. I need to open an issue for that one where I'll automate the creation of that install yaml file. That will certainly ease up the pain here. Let me know if it was that that you saw in this here.

Reply:
ok, it is clear now, thank you for the explanation.